util/diff/diff_test.go

```diff
@@ -297,3 +297,13 @@ func TestThreeWayDiffExplicitNamespace(t *testing.T) {
 	assert.Nil(t, err)
 	log.Println(ascii)
 }
+
+func TestUnmarshalDiffString(t *testing.T) {
+	JSONDiffStr := "{\n  \"spec\": {\n    \"ports\": {\n      \"0\": {\n        \"port\": [\n          81,\n          80\n        ],\n        \"targetPort\": [\n          79,\n          80\n        ]\n      },\n      \"_t\": \"a\"\n    }\n  }\n}\n"
```

Comment:
Can we put this in a more readable/maintainable constant?

Reply:
Fixed